### DISCUSSION TITLE
支付宝集五福多少三角面呢？

### DISCUSSION THREAD

**Discussion by ichenglixue** [galacean/discussion#58] [Engine]
支付宝集五福活动有多少三角面呢？  是通过高模烘焙然后给低模贴图的吗？  如果是的话，低模多少三角面呢？

**Comment by gz65555** [comment]
- 支付宝集五福活动有多少三角面呢？
  - 首页是 15000，游戏部分大概 7000，都是不是高模。
- 是通过高模烘焙然后给低模贴图的吗？
  - 是的。
- 如果是的话，低模多少三角面呢？
  - 实时渲染用的都是低模。

**Comment by gz65555** [comment]
@challengingcheng 问题我先关闭了，还有问题回复就好。
